README.md

```diff
@@ -329,12 +329,15 @@ This tool wraps underlying license / copyright scanners with a common API so all
 same way to easily run them and compare their results. If passed an ORT result file with an analyzer result (`-i`), the
 _scanner_ will automatically download the sources of the dependencies via the _downloader_ and scan them afterwards.
 
-Currently, the following license scanners are supported:
+We recommend to use ORT with one of the following scanners as their integration is tested best:
```

Comment:
Replace "is tested best:" with "has been thoroughly tested:"

Reply:
Done.